src/styles/main.scss

```diff
@@ -1,3 +1,7 @@
-body {
+@import "variables.scss";
+@import "../blocks/calendar.scss";
+
+:root {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
 }
```

Comment:
I believe "main.scss" folder should contain only imports, everything else should be moved to another files.